### PR TITLE
Tgbot 44 - Create object provider for Customer objects

### PR DESCRIPTION
**Pull request by yurasava**
https://schneidervs.atlassian.net/browse/TGBOT-44
Created a CustomerProvider class that exposes 5 customer objects.
There are also tests for the methods of the CustomerProvider class.

### CODE REVIEW

**Review by schneidervs — APPROVED** [review]

**Review by Sav4ykVladzislau — DISMISSED** [review]
Lgfm

File: stoservice/src/main/java/com/tutrit/stoservice/provider/CustomerProvider.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+import java.util.Set;
+
+public class CustomerProvider {
+
+    public Set<Customer> getCustomers() {
+        Set<Customer> result = new HashSet<>();
+        Customer initial = getInstance();
+
+        for (int i = 1; i <= 5; i++) {
+            result.add(new Customer(initial.getId().concat(String.valueOf(i)),
```
[the remark this answers is not in the document]
Comment (yurasava) [reply stoservice/src/main/java/com/tutrit/stoservice/provider/CustomerProvider.java:15]: This is the same

[the remark this answers is not in the document]
Comment (schneidervs) [reply stoservice/src/main/java/com/tutrit/stoservice/provider/CustomerProvider.java:15]: sorry.  this is my bad english.  I mean it's possible.

**Review by AlexanderYarokhovich — APPROVED** [review]
good

**Review by max-java — CHANGES_REQUESTED** [review]
please simplify provider logic

**Comment by schneidervs** [comment]
LGFM

**Review by schneidervs — APPROVED** [review]

**Review by Sav4ykVladzislau — APPROVED** [review]
lgfm

**Review by lamaconda — APPROVED** [review]
lgfm

**Pull request merged**